docs/addons/kasten-k10.md

```diff
@@ -0,0 +1,145 @@
+
+1. Create the IAM Role for Service Account for Kasten K10 pod to make API calls to AWS S3 and EC2 to backup and restore.
+2. Installs Kasten K10 in a new namespace "kasten-io".
+
+The IAM Policy is as follows:
+
+```
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:CopySnapshot",
+                "ec2:CreateSnapshot",
+                "ec2:CreateTags",
+                "ec2:CreateVolume",
+                "ec2:DeleteTags",
+                "ec2:DeleteVolume",
+                "ec2:DescribeSnapshotAttribute",
+                "ec2:ModifySnapshotAttribute",
+                "ec2:DescribeAvailabilityZones",
+                "ec2:DescribeSnapshots",
+                "ec2:DescribeTags",
+                "ec2:DescribeVolumeAttribute",
+                "ec2:DescribeVolumesModifications",
+                "ec2:DescribeVolumeStatus",
+                "ec2:DescribeVolumes"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "ec2:DeleteSnapshot",
+            "Resource": "*",
+            "Condition": {
+                "StringLike": {
+                    "ec2:ResourceTag/Name": "Kasten: Snapshot*"
+                }
+            }
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:CreateBucket",
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:PutBucketPolicy",
+                "s3:ListBucket",
+                "s3:DeleteObject",
+                "s3:DeleteBucketPolicy",
+                "s3:GetBucketLocation",
+                "s3:GetBucketPolicy"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+
+```
+
+
+## Validating the Install
+
+To validate that K10 has been installed properly, the following command can be run in K10's namespace (the install default is `kasten-io`) to watch for the status of all K10 pods:
+
+kubectl get pods --namespace kasten-io --watch
+
+It may take a couple of minutes for all pods to come up but all pods should ultimately display the status of `Running`.
+```
+kubectl get pods --namespace kasten-io
+NAMESPACE     NAME                                    READY   STATUS    RESTARTS   AGE
+kasten-io     aggregatedapis-svc-b45d98bb5-w54pr      1/1     Running   0          1m26s
+kasten-io     auth-svc-8549fc9c59-9c9fb               1/1     Running   0          1m26s
+kasten-io     catalog-svc-f64666fdf-5t5tv             2/2     Running   0          1m26s
+```
+
+In the unlikely scenario that pods that are stuck in any other state, please follow the [support](https://docs.kasten.io/latest/operating/support.html#support) documentation to debug further.
+
+### Validate Dashboard Access
+
+By default, the K10 dashboard will not be exposed externally. To establish a connection to it, use the following `kubectl` command to forward a local port to the K10 ingress port:
```

Comment:
Does it make sense to integrate with ingress, e.g if aws loadbalancer controller is installed (ALB Ingress) or nginx? How customers will typically use it?

Reply:
I'm following our default guidance in our documentation here which is to not expose the GUI by default. You can fully manage Kasten K10 via kubectl and API calls and no exposed GUI is necessarily required. We make it easy/simple to expose K10 via ingress and loadbalancer through options that are documented.